src/forms/StripePaymentForm/StripePaymentForm.js

```diff
@@ -236,6 +282,7 @@ class StripePaymentForm extends Component {
         />
         {hasCardError ? <span className={css.error}>{this.state.error}</span> : null}
         {hasSubmitErrors ? <span className={css.error}>{submitErrorMessage}</span> : null}
+        {billingDetails}
```

Comment:
Thinking out loud: I'm not sure if those submit errors, should be below the billing details for above them.